partitioner/src/main/java/io/confluent/connect/storage/partitioner/DailyPartitioner.java

```diff
@@ -30,20 +30,26 @@
 
   @Override
   public void configure(Map<String, Object> config) {
-    long partitionDurationMs = TimeUnit.HOURS.toMillis(24);
+    final long partitionDurationMs = TimeUnit.HOURS.toMillis(24);
```

Comment:
I see. Fair point. Closer is better then. Because there's nothing special to this variable compared to other ones in this method.